main.js

```diff
@@ -264,6 +324,18 @@
 
                     const latLong = dataSource.latLong.map((fieldName) => record[fieldName]);
                     const latLongNoNulls = latLong.some((field) => !!field);
+
+                    //Prune to last 30 days
+                    if (record.incidentYear) {
```

Comment:
This is a great check. I was just thinking about this earlier, and we should probably prune in the actual SQL qeury, but this is good for now. Could you add a comment:
`TODO: prune in the SQL query instead` to remind us?